src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/task/ContainerRestrictions.java

```diff
@@ -28,25 +28,29 @@
 
   private int ramLimitMegabytes;
 
+  private int outputLimitBytes;
+
   private boolean networkDisabled;
 
   @JsonCreator
   public ContainerRestrictions(
       @JsonProperty("timeoutSec") int timeoutSec,
       @JsonProperty("diskWriteLimitMegabytes") int diskWriteLimitMegabytes,
       @JsonProperty("ramLimitMegabytes") int ramLimitMegabytes,
+      @JsonProperty("outputLimitBytes") int outputLimitBytes,
```

Comment:
What do you think about using Mb here - I doubt we are ever going to want to set this super aggressively?

Reply:
Mb might be better. (It's slightly confusing anyway, as depending on the Java version it might be 1 or 2 bytes per character... Not sure if there is a way to tell easily as it is an implementation detail of the JVM.)
I could do Megachars? Although that might be odd...

Reply:
Really??? I thought .length() would always give the number of UTF-16 code units in the string which is twice the number of bytes? Can you point me at where you saw different? (My world is rocked).

Reply:
Oh thaat. Yikes. I forgot about that. Let's go with outputLimitMegachars as you suggest. Its odd, but accurate, and reduces the chance of order of magnitude errors from silly-large numbers

Reply:
Okay; I've gone for kilochars since even one megachar is quite a lot of chars and I wanted to impose a 100kC limit on output by default. Hope that makes sense.